index.html

```diff
@@ -229,6 +207,27 @@ <h2>
               </ul>
           </div>
         </div>
+
+        <section class="contact-me">
+          <h3 class="contact-me-header">
+           I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h3>
+          <form action="#" method="post" class="form-contact-me">
+            <input type="text"  name="username" placeholder="Full name" id="name" required>
+            <input type="email" name="email" placeholder="Full name" id="email" required>
+            <textarea name="comment" id="text-area" cols="30" rows="10">Enter text here</textarea>
```

Comment:
- [ ] According to the requirements a user cannot submit the form if one or more of the fields is empty. Please add the `required` attribute also in the message field so it will prevent the form from being submitted if the message field is empty.